app/models/comments.rb

```diff
@@ -0,0 +1,7 @@
+class Comments < ApplicationRecord
+  belongs_to :posts, :users
+
+  def update_commment_count
+    posts.comments_counter += 1
+  end
```

Comment:
## Optional
- [ ] I suggest that you use [Rails active callbacks](https://guides.rubyonrails.org/active_record_callbacks.html), `after_save` to call the #update_comments_count method every time a new comment has been saved to database, then make the update_comments_count method private